api-reference/endpoint/genAIEvaluate.mdx

```diff
@@ -0,0 +1,6 @@
+---
+title: 'GenAI: Evaluate'
+openapi: 'POST /genAI/v1/evaluate'
```

Comment:
I'm so sorry, I updated this to be "perform" instead of "evaluate". Could we change this? Thank you!